api-tests/features/ticf/ticf-failed-journeys-error-scenarios.feature

```diff
@@ -0,0 +1,115 @@
+Feature: TICF failed journeys error scenarios
+
+  Rule: Via alternate doc route
+    Background: Start TICF alternate doc journey given user already has an existing TICF record
+      Given there is an existing TICF record for the user with details
+        | responseDelay | 0                            |
+        | type          | RiskAssessment               |
+        | txn           | randomUuid                   |
+      When I start a new 'medium-confidence' journey with feature set 'ticfCriBeta'
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I get an 'access_denied' OAuth error from the CRI stub
+      Then I get a 'page-multiple-doc-check' page response
+      When I submit a 'ukPassport' event
+      Then I get a 'ukPassport' CRI response
+      When I submit 'kenneth-passport-valid' details to the CRI stub
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+      When I submit 'kenneth-score-2' details to the CRI stub
+      Then I get a 'page-pre-experian-kbv-transition' page response
+      When I submit a 'next' event
+      Then I get a 'kbv' CRI response
+      When I submit 'kenneth-needs-enhanced-verification' details to the CRI stub
+      Then I get a 'pyi-suggest-other-options' page response
+
+    Scenario: TICF failed alternate doc journey - PYI_NO_MATCH
+      When I submit a 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I submit 'kenneth-passport-invalid' details to the CRI stub
+      Then I get a 'pyi-no-match' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P0' identity with a 'TICF' VC
+      And the TICF VC has properties
+        | cis  |                              |
+        | type | RiskAssessment               |
+        | txn  | randomUuid                   |
```

Comment:
Have removed the type check but since this is an assertion step would it be better to explicit about what we expect instead of having defaults? E.g. when the scenarios specify no CI journeys

Reply:
Actually ignore this, it reduces a lot of duplication and can still specify certain properties if need be

Reply:
Oh of course this is an assertion, not the setup. Never mind then - I agree